### DISCUSSION TITLE
ZepetoWorldMultiplay 스크립트를 찾을수가 없습니다.

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#660] [5. Q&A]
ZepetoWorldMultiplay 스크립트가 프로젝트에 없습니다.

패키지 버전을 낮춰도 없는데 혹시 제가 못찾고 있는걸까요 ? ㅠㅠ..

![image](https://user-images.githubusercontent.com/25768804/165203027-9f906e93-c57a-49ba-98ec-18401946081d.png)

**Comment by xowjd913** [comment]
패키지 버전 1.1.6 까지 내렸다가 다시 올리니까 정상적으로 생겼습니다.

**Comment by Zepeto-TechEvangelist** [comment]
ASSETS > Reimport All 을 사용하시면 최신 버전에서도 가능합니다.
